datasets/irs990.yaml

```diff
@@ -51,4 +51,8 @@ DataAtWork:
       URL: https://www.grantmakers.io
       AuthorName: Chad Kruse
       AuthorURL: https://www.chadkruse.com/
+    - Title: npo_classifier: Automated coding using machine-learning and remapping the U.S. nonprofit sector
```

Comment:
You just need to add quotes around the title text.

Reply:
Should be good now.